scripts/diff.py

```diff
@@ -78,6 +78,8 @@ def get_argparser():
 
         outputiidm_result_paths = list()
         outputiidm_result_paths.append(os.path.realpath(os.path.join(results_root, "outputs/finalState/outputIIDM.xml")))
+        if options.output_zip:
+            outputiidm_result_paths.append(os.path.realpath(os.path.join(results_root, "output/outputs/finalState/outputIIDM.xml")))
```

Comment:
It would be cleaner to update result_root instead of adding if for every type of files